package.json

```diff
@@ -17,7 +17,7 @@
     "build": "./scripts/build.sh",
     "build:watch": "./scripts/build_watch.sh",
     "start": "yarn --cwd docs start",
-    "extract-intl": "formatjs extract --format=./packages/i18n/transifex-transformer.js --out-file=./packages/i18n/data/core.json '{packages,src/components}/**/messages.js'",
+    "extract-intl": "formatjs extract --format=./packages/i18n/transifex-transformer.js --out-file=./packages/i18n/data/core.json '{packages,src/components}/**/messages.ts'",
```

Comment:
This change is not related, but I noticed the command was wiping out all translations when I tried to use it to check the other changes.